docs/reference/operator_parameters.md

```diff
@@ -252,8 +252,15 @@ configuration they are grouped under the `kubernetes` key.
   teams API. The default is `postgresql-operator`.
 
 * **infrastructure_roles_secret_name**
-  namespaced name of the secret containing infrastructure roles names and
-  passwords.
+  *deprecated*: namespaced name of the secret containing infrastructure roles
+  names and passwords.
```

Comment:
might be a minor thing, but should be like roles, names, and passwords, or is it role names and passwords?